theme/ItaliaTheme/Blocks/_argumentsInEvidence.scss

```diff
@@ -12,6 +12,18 @@
     background-color: $primary;
   }
 
+  .icon-container {
```

Comment:
farei una classe unica, tipo .icon-argument, cosi la riutilizzi anche nell'header dell'argomento e non riscriviamo gli stili due volte. 
Metterei anche come colore dell'icona il $primary. Guardando gli sketch di agid e gli altri che ha fatto Serena coincide sempre con il colore primario, solamente reggio usa il $tertiary, quindi sovrascriverei solo per reggio poi questo stile.

Reply:
può essere utile creare proprio un componente per l'icona dell'argomento in modo da non duplicare neanche la parte html? se si qual'è secondo te la posizione migliore per creare questi componenti "comuni" a tutta l'applicazione in volto?

Reply:
sotto View/Commons/Argument? oppure View/Commons